requirements/test.txt

```diff
@@ -1,16 +1,22 @@
 -r base.txt
+
+# These package versions must be kept in sync with edx-platform as much as possible.
 django>=1.11.27,<1.12
 celery>=3.1.25,<4.0
+xblock-utils>=1.2.0,<1.3
+jsonfield==2.0.2
```

Comment:
Since the _package_ itself does depend on `jsonfield`, and not just the unit/integration tests, this also requires an entry in the `install_requires` list in `setup.py`. Presumably like:

```python
    install_requires=[
        'apscheduler',
        'google-api-python-client',
        'jsonfield>=2.0.2,<3.0',
        'paramiko',
        'python-heatclient',
        'python-keystoneclient',
        'python-novaclient',
    ],
```

Reply:
No, we're not doing that anymore.  Package dependencies that come via edX are not to be specified explicitly, so as to avoid runtime conflicts when edX pops an upgrade.

Reply:
(Except on test requirements, of course.  Instead of depending on edx-platform itself, it is much more efficient to duplicate the dependencies edx-platform introduces and on which the XBlock depends.)